app.config.ts

```diff
@@ -0,0 +1,21 @@
+const appName = 'Kuba Mobile';
```

Comment:
This file must be removed from the repository (since it is in the .gitignore) !

But otherwise, you could keep a shared configuration file (so in the repository) for what is ... shareable (for example the name of the application). And use environment variables for what is specific, like the API_URL.

For this, you can have a look at https://www.npmjs.com/package/react-native-dotenv

Personally, I prefer the second solution which is closer to good practice from [twelve-factor](https://12factor.net/config)